.ci_support/linux_64_cuda_compiler_version10.1.yaml

```diff
@@ -1,28 +0,0 @@
-c_compiler:
-- gcc
-c_compiler_version:
-- '7'
-cdt_name:
-- cos6
-channel_sources:
-- conda-forge,defaults
-channel_targets:
-- conda-forge main
-cuda_compiler:
-- nvcc
-cuda_compiler_version:
-- '10.1'
```

Comment:
It appears CUDA 10.1  is getting dropped. Not sure if we were trying to keep this here or if it’s ok to remove